src/function.ts

```diff
@@ -138,6 +141,25 @@ abstract class FunctionBase<P, O> implements IFunction<P, O> {
         return context.json(request);
       },
     };
+
+    this.apiGWVtl = {
+      renderRequest: (call, context) => {
+        const payloadArg = call.getArgument("payload");
+        const payload = payloadArg?.expr
+          ? context.eval(payloadArg.expr)
+          : "$null";
+        return context.json(payload);
+      },
+
+      createIntegration: (options) => {
+        this.resource.grantInvoke(options.credentialsRole);
+        return new aws_apigateway.LambdaIntegration(this.resource, {
+          ...options,
+          proxy: false,
+          passthroughBehavior: aws_apigateway.PassthroughBehavior.NEVER,
+        });
```

Comment:
I'm not liking how the hooks for each integration type are so different. They are very different but im surprised we don't have patterns.